### PR TITLE
Releasenotes 1.9.0

### PR DESCRIPTION
**Pull request by stayallive**
Release notes and doc updates for 1.9.0.

> https://github.com/getsentry/sentry-php/compare/1.8.4...master

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,17 @@
 # CHANGELOG
 
-## Unreleased
```
Comment (Jean85) [inline CHANGELOG.md]: Please leave the `unreleased` header at the top.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,17 @@
...
-
-- ...
+## 1.9.0 (2018-05-03)
+
+- Fixed undefined variable (#588)
+- Fix for exceptions throwing exceptions when setting event id (#587)
+- Fix monolog handler not accepting Throwable (#586)
+- Add `excluded_exceptions` option to exclude exceptions and their extending exceptions (#583)
```
Comment (Jean85) [inline CHANGELOG.md]: I would add here that is a 2.x forward compat functionality.

File: docs/config.rst [hunk 3]
```diff
@@ -241,6 +241,26 @@ The following settings are available for the client:
...
+
+    The timeout for sending requests to the Sentry server in seconds, default is 2 seconds.
+
+    .. code-block:: php
+
+        'timeout' => 2,
+
+.. describe:: excluded_exceptions
+
+    Exception that should not be reported, exceptions extending exceptions in this list will also
```
Comment (Jean85) [inline docs/config.rst]: Here too, so it will be clear.

File: CHANGELOG.md (hunk 2 above)
Comment (stayallive) [reply CHANGELOG.md]: I don't think we should do that yet. We don't know how and if when we release (bit of expectation management), when we do we can create the upgrade guide with all options that are forward compatible and which need removing / updating.

Notice that `excluded` is currently not documented for `1.x` and I intend to leave it that way (but ofcourse do mention it in the upgrade guide later).

**Pull request merged**